src/Microsoft.Azure.SignalR/HubHost/ServiceLifetimeManager.cs

```diff
@@ -27,10 +27,13 @@ internal class ServiceLifetimeManager<THub> : ServiceLifetimeManagerBase<THub> w
             ILogger<ServiceLifetimeManager<THub>> logger, AzureSignalRMarkerService marker)
             : base(serviceConnectionManager, protocolResolver)
         {
+            // after core 3.0 UseAzureSignalR() is not required.
+#if !NETCOREAPP3_0
```

Comment:
I'd recommend changing this so there is a compile error in future TFM changes

https://github.com/aspnet/AspNetCore/blob/648bdf706e823bab9f244f3a06d378de6961f221/src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/WebSocketsTransport.cs#L234